src/debugger.ts

```diff
@@ -0,0 +1,55 @@
+import * as vscode from "vscode"
+import {CancellationToken, DebugConfiguration, Disposable, ProviderResult, WorkspaceFolder} from "vscode"
+import {number} from "vscode-languageclient/lib/utils/is";
+
+export const runScala = "metals-scala-run";
+export const startSession = "debug-session-start";
+const configurationType = "scala";
+
+export async function run(buildTarget: BuildTargetIdentifier, mainClass: string): Promise<Boolean> {
```

Comment:
Is it possible to automatically register a "launch configuration" if it doesn't exist before? https://code.visualstudio.com/docs/editor/debugging#_launch-configurations This would make it possible to trigger the run via keyboard shortcut after clicking on "Run" for the first time.

Reply:
This can be left for a separate PR.